tests/simple_tests.spec.js

```diff
@@ -0,0 +1,41 @@
+const { test, expect } = require('@playwright/test');
+
+test.describe("Simple tests", () =>{
```

Comment:
Let's name it properly, smth like "adding product to cart for authorized user" or smth like that